resources/js/store/product.js

```diff
@@ -46,6 +57,36 @@ const actions = {
             commit('setProduct', []);
         }
     },
+
+    async addToWishList({ commit}, payload) {
+        try {
+            const response = await axios.post('/wishList/'+payload.product, {type:payload.type});
+
+        } catch (error) {
+
+        }
```

Comment:
Як варіант, замість try/catch:
```suggestion
            axios.post('/wishList/'+payload.product, {type:payload.type})
                .then((response) => {})
                .catch((error) => {})
```